docs/simreal/simwithreal.rst

```diff
@@ -0,0 +1,64 @@
+.. _`Switching between the Real Robot and the Simulated Robot in this Simulation`:
```

Comment:
This label is a bit long.  Actually, you don't really need labels at the top of the file.  If you want to reference the whole page, you can use

    :doc:`filename`

Reply:
thank you :)